cmd/prombench/README-prombench.md

```diff
@@ -0,0 +1,124 @@
+# Automated Prometheus E2E testing and benchmarking.
```

Comment:
why is it called README-prombench.md and not just README.md?

Also for git to detect and show a file as moved you should use
git mv README.md cmd/prombench/README.md

or `git add -A .` , this will detect when a file was move.

This makes reviews a lot easyer, knowing that the file was not modified and just moved, I don't have to read it to see if there any changes there.

Reply:
The `README-prombench.md` file contains some text, and it was previously in the prombench folder as the default `README.md`. I am not sure who created it, but before deleting I wanted to ask whether it should be kept or deleted. So, what should we do about it?

Thanks for the suggestion, I will use it next time.

Reply:
> Thanks for the suggestion, I will use it next time.

you need to use for this time :smile: 

Otherwise I can't read the PR properly.

> The README-prombench.md file contains some text, and it was previously in the prombench folder as the default README.md. I am not sure who created it, but before deleting I wanted to ask whether it should be kept or deleted. So, what should we do about it?

combine the two and just name it as README.md

Reply:
How can I move it since it is already moved? Do I have to move it back, remove the current `README.md`, commit the changes, moved it to `prombench folder againg, add the new `README.md` again, and commit? Would that work?

Reply:
Yes need to do it from the begining.